packages/api/src/routes/middlewares/propelauth.ts

```diff
@@ -29,6 +31,20 @@ export function getAuth(): PropelAuth | undefined {
   return auth;
 }
 
+export async function getCxIdFromJwt(req: Request): Promise<string> {
+  const auth = getAuth();
+
+  const jwtStr = req.header("Authorization");
+  if (!jwtStr) throw new ForbiddenError();
+
+  const user = await auth.validateAccessTokenAndGetUser(jwtStr);
+
+  const cxId = getCxId(user);
+  if (!cxId) throw new ForbiddenError();
+
+  return cxId;
+}
+
```

Comment:
why do we need to use propelauth just to get the cx id from the JWT? The api gw actually validates the jwt, right?